Automattic/wp-calypso
# Untangling: Add Advanced Tools -> SFTP/SSH

Fixes:

- https://github.com/Automattic/dotcom-forge/issues/9619

## Proposed Changes

This PR adds the Advanced Tools -> SFTP/SSH subtab under the `untangling/hosting-menu` flag.

|<img width="830" alt="image" src="https://github.com/user-attachments/assets/13a8eccc-f35b-4daf-ab16-4cb7a8fb2ce3">|<img width="799" alt="image" src="https://github.com/user-attachments/assets/d79d0ebb-729b-4173-a5c2-c3394e330961">|
|-|-|

## Why are these changes being made?

pbxlJb-6ye-p2

## Testing Instructions

1. Go to /sites
2. Click an Atomic site
3. Go to Advanced Tools -> SFTP/SSH subtab.
4. Verify that this tool is working.
5. Regression testing:
    - Go to Server Settings
    - Verify that the original SFTP/SSH card is still working.

> [!WARNING]  
> Practically we can't test whether the title should be `SFTP` or `SFTP/SSH`, because the only sites that have SFTP but not SSH are those on Pro plan, which has been cancelled.

## Pre-merge Checklist
